webpack/assets.js

```diff
@@ -0,0 +1,42 @@
+import WebpackIsomorphicToolsPlugin from 'webpack-isomorphic-tools/plugin';
+
+export default {
+  assets: {
+    images: {
+      extensions: ['jpeg', 'jpg', 'png', 'gif'],
+      parser: WebpackIsomorphicToolsPlugin.url_loader_parser
+    },
+    fonts: {
+      extensions: ['woff', 'woff2', 'ttf', 'eot'],
+      parser: WebpackIsomorphicToolsPlugin.url_loader_parser
+    },
+    svg: {
+      extension: 'svg',
+      parser: WebpackIsomorphicToolsPlugin.url_loader_parser
+    },
+    styles: {
+      extensions: ['css', 'less', 'sass', 'scss', 'styl'],
+      filter: (module, regex, options, log) => {
```

Comment:
We can also write it like this:

```js
styles: {
  filter(module, regex, options, log) {
    // ...
  },
  // ...
}
```

Which is I think closer to the syntax that we are using for `class`es and a bit shorter.